### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [shlusiak/Freebloks-Android#21]
The contrast ratio between text and background should be at least 4.5 . The following image shows some elements that don't respect the contrast ratio. 

![IMG_20201117_152122](https://user-images.githubusercontent.com/25429624/99409327-8a73b400-28f1-11eb-812c-226c442b26ae.png)

> **How to reproduce the error**

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Freebloks](https://play.google.com/store/apps/details?id=de.saschahlusiak.freebloks)
3. Activate the registration from the Accessibility Scanner and use the Freebloks app.
4. After using the app, stop the recording.

The results will show the errors found in the app. For example:

![IMG_20201117_152052](https://user-images.githubusercontent.com/25429624/99409408-9e1f1a80-28f1-11eb-90b9-aef09059056d.png)
